base/applications/taskmgr/perfdata.c

```diff
@@ -125,14 +128,22 @@ static void SidToUserName(PSID Sid, LPWSTR szBuffer, DWORD BufferSize)
 VOID
 WINAPI
 CachedGetUserFromSid(
```

Comment:
On Windows, `CachedGetUserFromSid` is a function exported from `utildll.dll`. Can you write a test that shows how it behaves with a NULL Sid? Thanks.

Reply:
Thanks for the information.
Done: #2954.

Reply:
> `utildll.dll`

Shall we also start creating this dll? And plan to move `CachedGetUserFromSid()` from here there?